packages/ui/src/global.css

```diff
@@ -95,3 +95,7 @@
     --radius: 0.5rem;
   }
 }
+
```

Comment:
@Omri-Levy What do you think about this addition? It's purpose is to solved a UI issue in the DatePicker component that looks like a z-index issue, please talk with Matan to figure out if there's a different issue that needs to be solved